### PR TITLE
Update CI setting to follow ruby 2.4.1

### PR DESCRIPTION
**Pull request by kachick**
Feel free to merge or pend~ 😗

### CODE REVIEW

File: spec/sbpayment/crypto_spec.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 describe Sbpayment::Crypto do
   let(:key)  { SecureRandom.hex 12 }
-  let(:iv)   { SecureRandom.hex  8 }
+  let(:iv)   { SecureRandom.hex  4 }
```
Comment (banyan) [inline spec/sbpayment/crypto_spec.rb]: 📝 @kachick taught me this comes from openssl after upgrading to 2.4.0: https://github.com/ruby/openssl

**Comment by banyan** [comment]
👍

File: spec/sbpayment/crypto_spec.rb (hunk 1 above)
Comment (kachick) [inline spec/sbpayment/crypto_spec.rb]: Yay, I guess it is applied in https://github.com/ruby/ruby/commit/ce635262f53b760284d56bb1027baebaaec175d1 👁

**Comment by kachick** [comment]
Thank you!

**Pull request merged**